beta-src/src/components/svgr-components/WDPhaseArrowIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import ScrollButtonState from "../../enums/ScrollButton";
+
+interface phaseArrowProps {
+  direction: ScrollButtonState;
+  disabled?: boolean;
+}
+
+const WDPhaseArrowIcon: React.FC<phaseArrowProps> = function ({
+  disabled,
+  direction,
+}): React.ReactElement {
+  return (
+    <svg fill="none" height={15} width={8} xmlns="http://www.w3.org/2000/svg">
```

Comment:
height and width can just be strings if we aren't changing them

Reply:
Changed!